com.deflemask.DefleMask.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=DefleMask
+GenericName=Chiptune Tracker
+Comment=The multi-system Chiptune Tracker
+Exec=/app/share/DefleMask/DefleMask
```

Comment:
Probably worth creating a link to /bin & removing the prefix from here and the command attribute of the manifest.

Reply:
That would require me to write a shell wrapper since the DefleMask binary seems to care about where it is located

Reply:
nevermind, got it, updating to metainfo

Reply:
Let's keep it unresolved until it's pushed out, unless it's impossible without a wrapper.

Reply:
I have tested symlinking the binary to `/app/bin` and it didn't work

Reply:
also sorry for the "nevermind" comment it was for another review

Reply:
:+1: